stat.go

```diff
@@ -116,10 +112,9 @@ func ChiSquare(obs, exp []float64) float64 {
 	var result float64
 	for i, a := range obs {
 		b := exp[i]
-		if a == 0 && b == 0 {
```

Comment:
Unrelated to this PR: is this condition correct? When `a != 0` and `b ==0`, then we get division by zero.